definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2958,5 +2958,295 @@
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/co-respondent-generate-answers",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/co-respondent-received",
+    "RetriesTimeoutURLSubmittedEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "09/12/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "co-RespAOSReceivedFromAwaitingProcessServerService",
+    "Name": "AOS Received (Co-Respondent)",
+    "Description": "AOS Received (Co-Respondent)",
+    "DisplayOrder": 3,
+    "PreConditionState(s)": "AwaitingProcessServerService",
+    "PostConditionState": "AwaitingProcessServerService",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/co-respondent-generate-answers",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/co-respondent-received",
+    "RetriesTimeoutURLSubmittedEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "09/12/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "co-RespAOSReceivedFromAwaitingDWPResponse",
+    "Name": "AOS Received (Co-Respondent)",
+    "Description": "AOS Received (Co-Respondent)",
+    "DisplayOrder": 3,
+    "PreConditionState(s)": "AwaitingDWPResponse",
+    "PostConditionState": "AwaitingDWPResponse",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/co-respondent-generate-answers",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/co-respondent-received",
+    "RetriesTimeoutURLSubmittedEvent": "120,120",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "15/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "aosSubmittedUndefended",
+    "Name": "AOS Received (undefended)",
+    "Description": "AOS Received (undefended)",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved;AwaitingAlternativeService;AwaitingProcessServerService;AwaitingDWPResponse",
+    "PostConditionState": "AwaitingDecreeNisi",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/aos-received",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/aos-submitted",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "15/09/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "aosSubmittedDefended",
+    "Name": "AOS Submitted (defending)",
+    "Description": "AOS Submitted (defending)",
+    "DisplayOrder": 2,
+    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved;AwaitingAlternativeService;AwaitingProcessServerService;AwaitingDWPResponse",
+    "PostConditionState": "AosSubmittedAwaitingAnswer",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/aos-received",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/aos-submitted",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "15/09/2020",
```

Comment:
General question. Are we not updating the data of LiveFrom when we alter an event ?

Reply:
Well, they haven't been updated. They were moved from different file. This is a date when we defined it, not when it was moved to prod.

Obviously it's just an approach. Maybe we will change it one day.